### PR TITLE
Docstrings

### PR DESCRIPTION
**Pull request by bnmajor**
Add docstrings to the `ViewerRPC` and `Viewer` classes

### CODE REVIEW

File: itkwidgets/_type_aliases.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 Image = Union[np.ndarray, itkwasm.Image, zarr.Group]
 PointSet = Union[np.ndarray, itkwasm.PointSet, zarr.Group]
+CroppingPlanes = List[Dict[str, List[Union[float, int]]]]
```
Comment (thewtex) [inline itkwidgets/_type_aliases.py:19]: This is a nice addition.

We could make the types more self-documenting by using string `Literal`'s like in:

https://github.com/spatial-image/spatial-image/blob/main/spatial_image.py#L23

File: itkwidgets/viewer.py [hunk 2]
```diff
@@ -315,118 +456,313 @@ async def get_image(self, name: str = 'Image') -> NgffImage:
...
+
+        :return: The future for the coroutine, to be updated with the current
+        blend mode.
+        :rtype:  asyncio.Future | str
+        """
         return await self.viewer_rpc.itk_viewer.getImageBlendMode()
 
     @fetch_value
-    def set_image_color_map(self, colorMap: str):
+    def set_image_color_map(self, colorMap: str) -> None:
```
Comment (thewtex) [inline itkwidgets/viewer.py:479]: We should use snake_case for the arguments to keep the Python interfaces idiomatic Python when possible, i.e. colorMap -> color_map

**Review by thewtex — CHANGES_REQUESTED** [review]
Absolutely fantastic! :1st_place_medal: :book: 

A few minor requests inline.

File: itkwidgets/_type_aliases.py (hunk 1 above)
Comment (bnmajor) [reply itkwidgets/_type_aliases.py]: Awesome!

File: itkwidgets/viewer.py (hunk 2 above)
Comment (bnmajor) [reply itkwidgets/viewer.py]: Yes, makes sense, thanks!

**Review by thewtex — APPROVED** [review]
Awesome, thank you!!

Can we please rebase on `main`?

**Review by thewtex — APPROVED** [review]
:book:

**Pull request merged**